concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md

```diff
@@ -0,0 +1,60 @@
+The ColorBox accepts colors in hexadecimal (_"#FF0000"_), RGB (_"rgb(255, 0, 0)"_) and RGBA (_"rgba(255, 0, 0, 1)"_) formats, as well as <a href="http://www.w3schools.com/cssref/css_colors.asp" target="_blank">color names</a>. When users select a color from the drop-down editor, the textual representation of the selected color is hexadecimal.
```

Comment:
Don't forget about Oxford commas.

Representation. I hoped I'll never hear this from you: http://tw.devexpress.devx/language-guidelines/misused-words/represent/

Please do not forget that every "represent" in your text says of your negligence, laziness, or both.

```suggestion
The ColorBox accepts colors in hexadecimal (_"#FF0000"_), RGB (_"rgb(255, 0, 0)"_), and RGBA (_"rgba(255, 0, 0, 1)"_) formats, as well as <a href="http://www.w3schools.com/cssref/css_colors.asp" target="_blank">color names</a>. When users select a color from the drop-down editor, the ColorBox displays the selected color as a string in hexadecimal format.
```